amr-wind/wind_energy/actuator/ActuatorContainer.cpp

```diff
@@ -185,6 +199,46 @@ void ActuatorContainer::update_positions()
     m_is_scattered = true;
 }
 
+void ActuatorContainer::update_positions(const MeshMap* map)
+{
+    BL_PROFILE("amr-wind::actuator::ActuatorContainer::update_positions");
+    AMREX_ALWAYS_ASSERT(m_container_initialized && !m_is_scattered);
+
+    if (map == nullptr) {
+        // call the standard update if mapping is null
+        update_positions();
+    }
+
+    const auto dpos = gpu::device_view(m_data.position);
+    const auto* const dptr = dpos.data();
+    const int nlevels = m_mesh.finestLevel() + 1;
+    for (int lev = 0; lev < nlevels; ++lev) {
+        for (ParIterType pti(*this, lev); pti.isValid(); ++pti) {
+            const int np = pti.numParticles();
+            auto* pstruct = pti.GetArrayOfStructs()().data();
+
+            amrex::ParallelFor(np, [=] AMREX_GPU_DEVICE(const int ip) noexcept {
```

Comment:
The simplest thing to do to make this GPU compliant is to make this a non-device loop.  The code path is preserved for the standard actuator points to still execute on the gpu.